GoogleForCreators/web-stories-wp
# Media: Fix CORS proxy regression on canvas

## Context



There was a small bug with image hotlinking where the CORS proxy was not used in all places where it should have been

## Summary



Fixes an issue with hotlinked images on the canvas not being run through the CORS proxy.

Adds e2e tests to prevent regressions.

## Relevant Technical Choices



## To-do



## User-facing changes



Before:

![Screenshot 2022-04-29 at 15 28 44](https://user-images.githubusercontent.com/841956/165953840-c5d56696-d1b5-486a-9c74-d541ced622a5.png)

After:

![Screenshot 2022-04-29 at 15 47 46](https://user-images.githubusercontent.com/841956/165957362-79835c47-6002-4e37-9bf4-1601a4803ec0.png)


## Testing Instructions






This PR can be tested by following these steps:

1. Add the image `https://wp.stories.google/e2e-tests/example.jpg` in the editor via the "Insert by link" functionality
2. Verify that the image appears correctly on the canvas and also when previewing the story


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #11395
